css-multicol-2/Overview.bs

```diff
@@ -82,6 +82,48 @@ Stacking context</h3>
 
 	Issue: Add final content from previous level
 
+<h3 id=column-pseudo>
+Styling Columns: the ''::column'' pseudo-element</h3>
+
+The <dfn selector>::column</dfn> pseudo-element
+represents the individual columns
+in a [=multi-column container=].
+It only exists on [=multi-column containers=].
+
+A [=multi-column container=] has as many ''::column'' pseudo-elements
+as it has columns.
+They cannot be individually addressed;
+any styles applied to a multicol's ''::column'' pseudo-elements
+apply to all of them on that element.
+Each ''::column'' has the same size and position
+as its corresponding column:
+the same size as the column's [=available space=] in the [=inline axis=]
+and the container's [=content box=] in the [=block axis=].
+(It does not cover the gaps/rules between columns.)
+
+''::column'' pseudo-elements are treated as children of their [=multi-column container=],
+without any contents.
+They <em>do not</em> wrap the contents of the column,
+just fill the same space.
```

Comment:
I'm not sure this is true. I think we might reasonably say that they *do* wrap the contents of the column; and in the far future when we allow fragment-based styling, we might want to.

Reply:
We very specifically resolved that ::column was a leaf node, with no contents. See <https://github.com/w3c/csswg-drafts/issues/10715#issuecomment-2379924982> You even made some comments in support of that. ^_^